CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/recipes/")
 
 # Set default minimum C++ standard
 if(PARAVIEWO_TOPLEVEL_PROJECT)
-    set(CMAKE_CXX_STANDARD 17)
+    set(CMAKE_CXX_STANDARD 20)
```

Comment:
we dont need this here, right?

Reply:
eh, it was useful for making sure paraviewo was built in cpp20 - we could do this cleaner i guess (by only assigning it to the single paraviewo target), but it's a minor detail